accurest-core/src/test/groovy/io/codearte/accurest/builder/MockMvcSpockMethodBuilderSpec.groovy

```diff
@@ -669,4 +671,70 @@ class MockMvcSpockMethodBuilderSpec extends Specification implements WireMockStu
 		then:
 			spockTest.contains('''assertThatRejectionReasonIsNull(parsedJson.read('$.rejectionReason'))''')
 	}
+
+	def "should support inner map and list definitions"() {
+		given:
+
+			Pattern ONLY_ALPHA_UNICODE = Pattern.compile(/[\p{L}]*/)
```

Comment:
Would it be better to return Pattern instead of String?

It would be better to write ```$(stub(email())``` instead of ```$(stub(regex(email()))```

Or ```$(stub(emailRegex())``` ?

WDYT?